src/plugin/markers/index.js

```diff
@@ -94,6 +96,16 @@ export default class MarkersPlugin {
             window.addEventListener('resize', this._onResize, true);
             window.addEventListener('orientationchange', this._onResize, true);
             this.wavesurfer.on('zoom', this._onResize);
+
+            if (!this.markers.find(marker => marker.draggable)){
+                return;
+            }
+
+            this.mousemoveListener = (e) => this._onMouseMove(e);
+            window.addEventListener('mousemove', this.mousemoveListener);
```

Comment:
every `addListener` you add has to be `removeListener`-ed as well (in `destroy` method).

Reply:
Done